pyanaconda/core/constants.py

```diff
@@ -32,7 +32,6 @@
 # common string needs to be easy to change
 from pyanaconda import product
 productName = product.productName
```

Comment:
You can also delete this variable then?

Reply:
Not really. There are dependencies on this stuff all over the place, including initial setup. I filed a new task which details how to address all of this, but actually doing it needs to wait a bit.